### PR TITLE
"clear" alert first on submit

### PR DESCRIPTION
**Pull request by samsonasik**
without this, if more then one failed try, it will got many alert div like this : 
alert-multi

### CODE REVIEW

**Comment by acelaya** [comment]
Nice one! 
Thanks!

File: public/js/main.js [hunk 1]
```diff
@@ -15,6 +15,12 @@ var acelaya = {
 
         $form.submit(function (e) {
             e.preventDefault();
+
+            // remove .alert block when it exists
+            if ($(".container .alert").length) {
+                $(".container .alert").remove();
+            }
```
Comment (samsonasik) [inline public/js/main.js]: maybe more cool with : 

``` javascript
$('.container .alert').each(function() { $(this).length = 0 || $(this).remove(); });
```

?

Comment (acelaya) [inline public/js/main.js]: Wouldn't it be the same to just do this?

``` javascript
$(".container .alert").remove();
```

I think there is no need to check if the elemnt exists.

Comment (samsonasik) [inline public/js/main.js]: yeah, but the ".alert" element didn't created yet before form submitted.

Warm regards,

Abdul Malik Ikhsan

Pada 24 Jan 2015, pukul 18.35, Alejandro Celaya notifications@github.com menulis:

Comment (acelaya) [inline public/js/main.js]: Yes, but what I mean is that jQuery does not throw an error if provided selector is not found. It just does nothing, so there is no need to check that the element exists.

**Comment by samsonasik** [comment]
@acelaya ok, i've updated ;)

**Comment by acelaya** [comment]
@samsonasik thanks!

**Pull request merged**